components/wifi_connector/wifi_ap.cpp

```diff
@@ -0,0 +1,72 @@
+#include "wifi_ap.hpp"
+#include "event_handlers.hpp"
+
+WifiAP::WifiAP()
+{
+  cfg_ap_ = {};
+  cfg_ap_.ap = {};
+  cfg_ap_.ap.ssid_hidden = 0;
+  cfg_ap_.ap.authmode = WIFI_AUTH_WPA_WPA2_PSK;
+
+  this->max_connections(4);
```

Comment:
Is there a reason to limit the number of allowed connections to a AP?

If there is no need to be able to change it later, you can set it directly in the config above as well.
```c++
cfg_ap_.ap.max_connection = 4;
```

Then the whole maxconnections method can be removed I guess?